Recrutify/Recrutify.Services/Validators/BaseProjectValidator.cs

```diff
@@ -59,7 +59,10 @@ private void ConfigureRules()
                 .MustAsync(CheckPrimarySkillsAsync);
             RuleForEach(p => p.PrimarySkills)
                 .NotNull()
-                .NotEmpty();
+                .NotEmpty()
+                .ChildRules(x => x.RuleFor(x => x.TestLink)
+                .Must(LinkMustBeAUri)
```

Comment:
Put tab before it and before with message - it'S child rule, format correctly

Reply:
done